[2025-08-26T14:09:41.403+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.run_db_to_file_etl manual__2025-08-26T14:09:36.175372+00:00 [queued]>
[2025-08-26T14:09:41.423+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mamda_monthly_etl_pipeline.run_db_to_file_etl manual__2025-08-26T14:09:36.175372+00:00 [queued]>
[2025-08-26T14:09:41.424+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-08-26T14:09:41.453+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): run_db_to_file_etl> on 2025-08-26 14:09:36.175372+00:00
[2025-08-26T14:09:41.464+0000] {standard_task_runner.py:60} INFO - Started process 70 to run task
[2025-08-26T14:09:41.469+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mamda_monthly_etl_pipeline', 'run_db_to_file_etl', 'manual__2025-08-26T14:09:36.175372+00:00', '--job-id', '361', '--raw', '--subdir', 'DAGS_FOLDER/mamda_monthly_etl.py', '--cfg-path', '/tmp/tmp0ggtc2fk']
[2025-08-26T14:09:41.477+0000] {standard_task_runner.py:88} INFO - Job 361: Subtask run_db_to_file_etl
[2025-08-26T14:09:41.605+0000] {task_command.py:423} INFO - Running <TaskInstance: mamda_monthly_etl_pipeline.run_db_to_file_etl manual__2025-08-26T14:09:36.175372+00:00 [running]> on host 40a873810464
[2025-08-26T14:09:41.769+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mamda_etl' AIRFLOW_CTX_DAG_ID='mamda_monthly_etl_pipeline' AIRFLOW_CTX_TASK_ID='run_db_to_file_etl' AIRFLOW_CTX_EXECUTION_DATE='2025-08-26T14:09:36.175372+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-26T14:09:36.175372+00:00'
[2025-08-26T14:09:41.771+0000] {mamda_monthly_etl.py:93} INFO - === START: DB -> FILE ETL ===
[2025-08-26T14:09:41.772+0000] {mamda_monthly_etl.py:79} INFO - Subprocess ENV: DATA_DIR_FEE_PAYOUTS -> /mnt/mamda_data/fee_payouts
[2025-08-26T14:09:41.773+0000] {mamda_monthly_etl.py:80} INFO - Subprocess ENV: DATA_DIR_FEE_STATUS -> /mnt/mamda_data/fee_payouts_status
[2025-08-26T14:09:47.064+0000] {mamda_monthly_etl.py:107} INFO - DB->FILE STDOUT:
Utilisation du dossier externe
Utilisation du dossier externe
Utilisation du dossier externe
Utilisation du dossier externe

[2025-08-26T14:09:47.065+0000] {mamda_monthly_etl.py:109} WARNING - DB->FILE STDERR:
INFO:__main__:---- DÉMARRAGE DU PIPELINE DB -> FILE ----
INFO:__main__:--- Traitement pour base: sinauto_mcma_, type_aux: A ---
INFO:__main__:Données extraites avec succès pour sinauto_mcma_
INFO:__main__:sinauto_mcma_ : mapping réussi
INFO:__main__:sinauto_mcma_ : transformation réussie
ERROR:__main__:Erreur lors du traitement de sinauto_mcma_, type_aux=A : [Errno 13] Permission denied: '/mnt/mamda_data/fee_payouts/Avocats'
INFO:__main__:--- Traitement pour base: sinauto_mcma_, type_aux: E ---
INFO:__main__:Données extraites avec succès pour sinauto_mcma_
INFO:__main__:sinauto_mcma_ : mapping réussi
INFO:__main__:sinauto_mcma_ : transformation réussie
ERROR:__main__:Erreur lors du traitement de sinauto_mcma_, type_aux=E : [Errno 13] Permission denied: '/mnt/mamda_data/fee_payouts/Experts Automobiles'
INFO:__main__:--- Traitement pour base: sinauto_mcma_, type_aux: M ---
INFO:__main__:Données extraites avec succès pour sinauto_mcma_
INFO:__main__:sinauto_mcma_ : mapping réussi
INFO:__main__:sinauto_mcma_ : transformation réussie
ERROR:__main__:Erreur lors du traitement de sinauto_mcma_, type_aux=M : [Errno 13] Permission denied: '/mnt/mamda_data/fee_payouts/Medecins'
INFO:__main__:--- Traitement pour base: sinauto_mcma_, type_aux: C ---
INFO:__main__:Données extraites avec succès pour sinauto_mcma_
INFO:__main__:sinauto_mcma_ : mapping réussi
INFO:__main__:sinauto_mcma_ : transformation réussie
ERROR:__main__:Erreur lors du traitement de sinauto_mcma_, type_aux=C : [Errno 13] Permission denied: '/mnt/mamda_data/fee_payouts/Experts Comptables'
INFO:__main__:---- FIN DU PIPELINE DB -> FILE ----

[2025-08-26T14:09:47.066+0000] {mamda_monthly_etl.py:117} INFO - === DONE: DB -> FILE ETL ===
[2025-08-26T14:09:47.067+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-08-26T14:09:47.083+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=mamda_monthly_etl_pipeline, task_id=run_db_to_file_etl, execution_date=20250826T140936, start_date=20250826T140941, end_date=20250826T140947
[2025-08-26T14:09:47.154+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-26T14:09:47.193+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
